.htaccess

```diff
@@ -1,2 +1,3 @@
 RewriteEngine On
+RewriteBase /
```

Comment:
Where's this come from?

Reply:
oh shoot, I didn't add that to the index! Must have accidentally an `-a` :(